models/Config.yaml

```diff
@@ -40,7 +40,6 @@ x-examples:
     mqtt_server: homeassistant.lan
     mqtt_port: 1883
     mqtt_topic: openevse
-    mqtt_retained: false
```

Comment:
This is still valid because of the `new ConfigOptVirtualBool(flagsOpt, CONFIG_MQTT_RETAINED, 0, "mqtt_retained", "mrt"),`